Cargo.lock

```diff
@@ -619,9 +619,9 @@ dependencies = [
 [[package]]
 name = "chain-vote"
 version = "0.1.0"
-source = "git+https://github.com/input-output-hk/chain-libs.git?branch=master#c30099ab1c44eeb766ca98354f4fceca1199c793"
+source = "git+https://github.com/input-output-hk/chain-libs.git?branch=master#4d2c52ace7dc3d76e9c6bfc777e1d3fa6cd68053"
 dependencies = [
- "cfg-if 1.0.0",
+ "cfg-if 0.1.10",
```

Comment:
I find this a bit weird, but I guess it's not important

Reply:
I don't understand it, but it's changing pretty much every other update of Cargo.lock 🤷🏼‍♂️

Reply:
I wonder if it's really ok to use a `*` [here](https://github.com/input-output-hk/chain-libs/blob/43663b1a0661abcf1dab1ac1bb9cf4053727891d/chain-vote/Cargo.toml#L23). Not sure if it's related though. And as I said, I don't think it really makes any difference.

Reply:
> git blame

> ahhh that was me



probably just hacked it while trying out stuff because I didn't bother to look at what was the current version and then forgot to update it

Reply:
Hmm, need to double check it